lib/cdk-infra-stack.ts

```diff
@@ -345,35 +360,196 @@ export default class FhirWorksStack extends Stack {
             },
         };
 
+        const startExportJobLambdaFunctionDLQ = new Queue(this, 'startExportJobLambdaFunctionDLQ', {
+            retentionPeriod: Duration.days(14),
+            encryptionMasterKey: Alias.fromAliasName(this, 'startExportJobLambdaFunctionDLQKey', 'alias/aws/sqs'),
+        });
+        NagSuppressions.addResourceSuppressions(startExportJobLambdaFunctionDLQ, [
+            {
+                id: 'AwsSolutions-SQS3',
+                reason: 'This is a DLQ.',
+            },
+        ]);
+        const startExportJobLambdaFunctionDLQHttpsOnlyPolicy = new QueuePolicy(
+            this,
+            'startExportJobLambdaFunctionDLQHttpsOnlyPolicy',
+            {
+                queues: [startExportJobLambdaFunctionDLQ],
+            },
+        );
+        startExportJobLambdaFunctionDLQHttpsOnlyPolicy.document.addStatements(
+            new PolicyStatement({
+                effect: Effect.DENY,
+                actions: ['SQS:*'],
+                resources: [startExportJobLambdaFunctionDLQ.queueArn],
+                principals: [new AnyPrincipal()],
+                conditions: {
+                    Bool: {
+                        'aws:SecureTransport': false,
+                    },
+                },
+            }),
+        );
         const startExportJobLambdaFunction = new NodejsFunction(this, 'startExportJobLambdaFunction', {
             ...defaultBulkExportLambdaProps,
             handler: 'startExportJobHandler',
+            reservedConcurrentExecutions: isDev ? 10 : 200,
+            deadLetterQueue: startExportJobLambdaFunctionDLQ,
         });
 
+        const stopExportJobLambdaFunctionDLQ = new Queue(this, 'stopExportJobLambdaFunctionDLQ', {
```

Comment:
Is it possible (or even a good idea) for us to use the same DLQ for all the export lambdas?

Reply:
created!